dev/integration_tests/flutter_gallery/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -253,24 +252,6 @@
 			shellPath = /bin/sh;
 			shellScript = "/bin/sh \"$FLUTTER_ROOT/packages/flutter_tools/bin/xcode_backend.sh\" build";
 		};
-		9935C741A3A9624590A8630E /* [CP] Embed Pods Frameworks */ = {
```

Comment:
CocoaPods removed this since it's no longer embedding and code signing the Flutter.framework.